fresnel/geometry.py

```diff
@@ -182,42 +183,37 @@ def radius(self):
     def color(self):
         return util.array(self._geometry.getColorBuffer(), geom=self)
 
-class Prism(Geometry):
-    R""" Prism geometry.
+class Polygon(Geometry):
+    R""" Polygon geometry.
 
-    Define a set of right convex prism primitives. The bottom polygon face is always in the xy plane. Each prism may
-    have a different height and rotation angle.
+    Define a set of simple polygon primitives. Each polygon face is always in the xy plane. Each polygon may
+    have a different color and rotation angle.
 
     Args:
         scene (:py:class:`fresnel.Scene`): Add the geometry to this scene
-        vertices: The vertices of the polygon in a counter clockwise winding direction.
-          **Type:** anything convertible by numpy to a Nx2 array of floats.
-        position: Positions of the prisms, *optional*.
-          **Type:** anything convertible by numpy to a Nx3 array of floats.
-        height: Height of each prism in the z direction, *optional*.
-          **Type:** anything convertible by numpy to a N length array of floats.
-        angle: Rotation angle of each prism (in radians), *optional*.
-          **Type:** anything convertible by numpy to a N length array of floats.
-        color: (r,g,b) color of each particle, *optional*.
-          **Type:** anything convertible by numpy to a Nx3 array of floats.
-        N (int): Number of prisms in the geometry. If ``None``, determine ``N`` from ``position``.
+        vertices (`numpy.ndarray` or `array_like`): (``Nx2`` : ``float32``) - polygon vertices.
+        position (`numpy.ndarray` or `array_like`): (``Nx2`` : ``float32``) -  Position of each polygon.
+        angle (`numpy.ndarray` or `array_like`): (``N`` : ``float32``) -  Orientation angle of each polygon.
+        color (`numpy.ndarray` or `array_like`): (``Nx3`` : ``float32``) - Color of polygon.
+        rounding_radius (float): Rounding radius for spheropolygons
+        N (int): Number of polygons in the geometry. If ``None``, determine ``N`` from *points*.
```

Comment:
Should this say `from ``positions``.`?
(positions instead of points, code font instead of bolding?)